src/bournemouth/chat_service.py

```
@@ -26,6 +26,11 @@
 
     from .openrouter import ChatMessage, StreamChunk
 
+StreamFunc: typing.TypeAlias = typing.Callable[
+    ["OpenRouterService", str, list["ChatMessage"], str | None],
+    typing.AsyncIterator["StreamChunk"],
+]
```

Ruff (0.11.9)

Type alias `StreamFunc` uses `TypeAlias` annotation instead of the `type` keyword

Use the `type` keyword

(UP040)